app.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const expressLayouts = require('express-ejs-layouts');
+const bodyParser = require('body-parser');
```

Comment:
I think you forgot to do --save when you did npm install body-parser.  The reason you do this is to add it do the package.json so when I do `npm install` it will install all of it for me